tbx/project_styleguide/templates/patterns/molecules/blog-item/blog-item.yaml

```diff
@@ -7,8 +7,8 @@ context:
 
 tags:
   image:
-    item.author.image fill-100x100 class="avatar__image":
-      raw: '<img src="https://placekitten.com/100/100" class="avatar__image">'
+    item.author.image fill-100x100 format-webp:
+      raw: '<img src="https://placekitten.com/100/100">'
```

Comment:
Q: I can see there's other instances of placekitten used in the yaml - should we add a line about this to https://github.com/torchbox/torchbox.com/blob/integration/sprint-1/docs/front-end/placeholder-images.md ?

Reply:
I was wondering if we ought to switch to unsplash to be more uniform with our other builds actually. I don't think it would take too long and maybe I could do it as part of this MR.

Reply:
It would be cool if we could fetch the pattern library images with webp - it looks like you can get unsplash images with webp, but I'm not sure it's supported in the url fetching method we use (which I can't actually find documented any more). Example webp image: https://images.unsplash.com/photo-1461988320302-91bde64fc8e4?ixid=2yJhcHBfaWQiOjEyMDd9&&fm=webp&w=400&fit=max
But this fetches a jpeg: https://source.unsplash.com/143x143?fm=webp

Reply:
I've replaced the kittens with unsplash jpgs.